mixer/cmd/mixc/cmd/util.go

```diff
@@ -248,7 +248,7 @@ func decodeStatus(status rpc.Status) string {
 	return result
 }
 
-// nolint:deadcode
+/*
```

Comment:
This is commented out. If you want to keep it, a comment explaining why this should be commented out and preserved would help.